python/foglamp/plugins/south/dht11/dht11.py

```diff
@@ -25,19 +26,28 @@
 
 _DEFAULT_CONFIG = {
     'plugin': {
-         'description': 'Python module name of the plugin to load',
-         'type': 'string',
-         'default': 'dht11'
+        'description': 'Python module name of the plugin to load',
+        'type': 'string',
+        'default': 'dht11',
+        'readonly': 'true'
+    },
+    'assetName': {
```

Comment:
assetName or assetNamePrefix?

Reply:
The rule of thumb for assetNamePrefix and assetName is that we apply assetNamePrefix to those plugins where readings for multiple sensors are recorded separately (e.g. Sensortag, envirophat) and assetName for those where only one reading (even for multiple sensors) is recorded (e.g. dht11).